tools/src/main/scala/scala/scalanative/profiling/Events.scala

```diff
@@ -0,0 +1,16 @@
+package scala.scalanative
+package profiling
+
+sealed trait Event
+object Event {
+  case class Call(ptr: Long, argc: Int)            extends Event
+  case class Load(ptr: Long)                       extends Event
+  case class Store(ptr: Long)                      extends Event
```

Comment:
We should try to avoid unstable information like raw pointers in Load, Store, Call, As, Is, Unbox. Those are not stable from run to run and it's going to be quite hard to reason about them.